docs/docs/reference/Vectors.md

```diff
@@ -209,14 +272,19 @@ Optional parameters are:
 
 * `EF_RUNTIME` - The number of maximum top candidates to hold during the KNN search. Higher values of `EF_RUNTIME` will lead to a more accurate results on the expense of a longer runtime. Defaults to the `EF_RUNTIME` value passed on creation (which defaults to 10).
 
+* `EPSILON` - Relative factor that sets the boundaries in which a range query may search for candidates. That is, vector candidates whose distance from the query vector is `radius*(1 + EPSILON)` are potentially scanned, allowing more extensive search and more accurate results (on the expense of runtime). Defaults to the `EPSILON` value passed on creation (which defaults to 0.01).
+
 ## Query tips
```

Comment:
Seems more important than just tips.
Consider "Important Notes" or use a banner (as already being used in other places)
by wrapping it with
`{{% alert title="Notes" color="info" %}}`
and
`{{% /alert %}}`